exonum/src/helpers/fabric/details.rs

```diff
@@ -416,6 +471,29 @@ impl Command for GenerateNodeConfig {
                 "listen-address",
                 false,
             ),
+            Argument::new_named(
+                CONSENSUS_KEY_PATH,
+                false,
+                "Path to the file storing consensus private key (default: ./consensus.toml)",
+                "c",
+                "consensus-path",
+                false,
+            ),
+            Argument::new_named(
+                SERVICE_KEY_PATH,
+                false,
+                "Path to the file storing service private key (default: ./service.toml)",
+                "s",
+                "service-path",
+                false,
+            ),
+            Argument::new_flag(
+                NO_PASSWORD,
```

Comment:
What is the utility of this flag? I'm not an expert in designing secure UX, but I think that the *only* case where we might not ask for a password is creating a test network. We would want to provide domain separation (i.e., make it impossible to run a test network node in the "real" network, or vice versa; this might include different address format if/when address appear in Exonum) and otherwise make clear that a test network is not secure and should not be used in production.

Reply:
Sounds reasonable... But it's not quite clear to me what kind of interface we should provide instead